en/protocol/parameter.md

```diff
@@ -11,8 +68,187 @@ sequenceDiagram;
     Drone->>GCS: Send N parameters
     GCS->>GCS: Start receive timeout
     GCS->>Drone: Request dropped params with PARAM_REQUEST_READ
+{% endmermaid %}
+
+### Read Single Parameter
+
+A single parameter can be read by the [PARAM_REQUEST_READ](../messages/common.md#PARAM_REQUEST_READ) message.
+
 
+
+### Write Parameters
+
+As *QGroundControl* does not have its own list of the parameters on startup, before writing a parameter first the parameter list has to be read once. After that, parameters can be written individually by sending the key->value pair to the component. As QGroundControl keeps track of changed parameters, clicking the write button will only transmit those which have changed in value. The MAV **has to acknowledge the write operation** by emitting a [PARAM_REQUEST_READ](../messages/common.md#PARAM_VALUE)  value message with the newly written parameter value.
```

Comment:
@DonLakeFlyer Generally, can I replace QGroundControl with "GCS"? Since protocol docs should in theory be implementable by any GCS?

Reply:
I agree.

Reply:
I have done this partially. However some of this remains QGC specific. OK for now.